dts/bindings/clock/st,stm32-lse-clock.yaml

```diff
@@ -12,9 +12,11 @@ properties:
       type: int
       required: true
       description: |
-        LSE driving capability
+        LSE driving capability, within the range 0 to 3.
+        0 represents the lowests driving capability, 3
+        the highest.
```

Comment:
Thanks! Could you please squash this into the original commit which added this file? We don't do this sort of "fixup" commit in Zephyr pull requests.

Reply:
Squashed them all together, as they basically cover the same thing, just for different SoCs.